substrate/frame/support/procedural/src/benchmark.rs

```diff
@@ -419,6 +419,7 @@ pub fn benchmarks(
 	};
 
 	let krate = generate_crate_access_2018("frame-benchmarking")?;
+	let frame_system = generate_crate_access_2018("frame-system")?;
```

Comment:
A comment about what this does any why it's needed?

Reply:
This is just for unhardcoding `frame_system`, as moving forward this could be reexported from the `frame` crate